sdk/src/wallet/types/mod.rs

```diff
@@ -160,13 +163,13 @@ impl TryFromDto<TransactionWithMetadataDto> for TransactionWithMetadata {
             inclusion_state: dto.inclusion_state,
             timestamp: dto
                 .timestamp
-                .parse()
-                .map_err(|_| BlockError::InvalidField("timestamp"))?,
+                .parse::<u128>()
+                .map_err(|e| PayloadError::InvalidTimestamp(e.to_string()))?,
```

Comment:
Should we still do the string conversion or change the error to contain the actual `ParseIntError`?

Reply:
Maybe not https://github.com/iotaledger/iota-sdk/pull/2046#discussion_r1507370969 ? :trollface:

Reply:
It's fine to contain a 3rd party error, in my comment it's not even wrapped at all

Reply:
I don't think we should since these errors are more generic than just the int parsing